artichoke-backend/src/extn/core/integer/mod.rs

```diff
@@ -140,9 +143,12 @@ impl Integer {
                     Ok(vec![chr])
                 }
                 _ => {
-                    let mut message = String::new();
+                    let mut message = spinoso_string::String::new();
                     write!(&mut message, "{} out of char range", self.as_i64()).map_err(WriteError::from)?;
-                    Err(RangeError::from(message).into())
+                    match spinoso_string::String::alloc_value(message, interp) {
+                        Ok(msg) => Ok(msg.inspect(interp)),
+                        Err(_) => Err(RangeError::from(message).into()),
+                    }
```

Comment:
This code constructs a `Vec<u8>` which is passed directly to an exception constructor. It does not need to go through a spinoso string.